sdk/go/common/resource/plugin/langruntime_plugin.go

```diff
@@ -342,3 +343,58 @@ func (h *langhost) GetProgramDependencies(info ProgInfo, transitiveDependencies
 	logging.V(7).Infof("%s success: #versions=%d", prefix, len(results))
 	return results, nil
 }
+
+func (h *langhost) RunPlugin(info RunPluginInfo) (io.Reader, io.Reader, context.CancelFunc, error) {
```

Comment:
Bike shedding nit: do you envision this being used for analyzer or language plugins as well, or would calling this `RunProvider` make more sense?

Reply:
Yes I expect this to be used for analyzers and languages as well. (Languages is a funny one, might not allow that in the end because of the circularness of it)